racket-poppler/render-tex.rkt

```diff
@@ -44,15 +44,15 @@
   (set! defaultdir
         (or defaultdir
             (case (system-type)
-              [(macosx) "/Library/TeX/texbin"]
+              [(macosx) "/usr/texbin"]
```

Comment:
Hi,
I have both but not on the same system.

in my OS X 10.10 system I only have /usr/texbin
 - /Library/TeX/texbin does not exist (/Library/TeX/ does exist, but
pdflatex etc is buried a few levels down)
 - mactex 2014

in my OS X 10.11 system I have /Library/TeX/texbin
 - /Library/TeX/texbin is the default for texshop
 - /usr/texbin does not exist
 - mactex 2015

I'm lucky I've got an old laptop I can test with =)

s.


On Tue, Feb 23, 2016 at 5:06 PM Jens Axel Søgaard <notifications@github.com>
wrote:

> In racket-poppler/render-tex.rkt
> <https://github.com/soegaard/racket-poppler/pull/8#discussion_r53812913>:
>
> > @@ -44,15 +44,15 @@
> >    (set! defaultdir
> >          (or defaultdir
> >              (case (system-type)
> > -              [(macosx) "/Library/TeX/texbin"]
> > +              [(macosx) "/usr/texbin"]
>
> That is: Do you have both /use/texbin and /Library/TeX/texbin ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/soegaard/racket-poppler/pull/8/files#r53812913>.
>